gh_index_issues_to_dts_portal.py

```diff
@@ -78,30 +93,65 @@ def build_payload(project_records, project_issues):
     for issue in project_issues:  # iterate over gh issues
         pipeline = find_pipeline_by_issue(zenhub_metadata, issue.number)
 
+        last_comment_body = None
+        last_comment_date = None
+        comments = issue.get_comments()
+        # an issue often has more than one assignee, this returns the list of users assigned to the issue
+        assignees = issue.assignees
+        assignees_logins = [user.login for user in assignees]
+        assignees_string = " ".join(assignees_logins)
```

Comment:
would it be preferable to comma-separate these? or newline (`\n`) them?

Reply:
Nbd, but newline would be nice.

Reply:
I added a comma. I tried a newline, but the formatting didnt come through in knack, i think its a limitation of the short text field